awx/main/tests/live/pytest.ini

```diff
@@ -0,0 +1,3 @@
+# This file is needed to undo the pytest settings from the project root
+[pytest]
+addopts =  -p no:django -p awx.main.tests.live.pytest_django_config
```

Comment:
what unsets `DJANGO_SETTINGS_MODULE` from awx/pytest.ini? is that what "no:django" does?

Reply:
IIRC, it only respects the `pytest.ini` from the current directory, and you can only run these from the `awx/main/tests/live` directory. So the `pytest.ini` from a parent folder won't even be considered.

Using `-p no:django` disables the plugin "django", which has `pytest_` prefixed, so the full plugin name is `pytest_django`. So this disables the pytest-django, it is installed as a library but pytest doesn't load it as a plugin.

Reply:
ah okay thanks for the explanation